src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentSplitForm/IncidentSplitForm.tsx

```diff
@@ -67,7 +68,7 @@ const IncidentSplitForm: FC<IncidentSplitFormProps> = ({
   onSubmit,
   isSubmitting,
 }) => {
-  const maxNoteLength = 1000
+  const maxNoteLength = 3000
```

Comment:
This is still 1000 right?

Reply:
[The design](https://www.figma.com/file/HCZLFhdGNf6s6NnQeZYuKZ/Schetsen?type=design&node-id=904-1080&mode=design&t=pEjc6a3yGcU7OHfH-0) specifies 3000, I figured that was what the acc criterion 'aantal tekens in text area' meant?